tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -70,22 +72,40 @@ def test_ebs_snapshot(
 
 
 # cn-north-1 does not support KMS
-@pytest.mark.dimensions("ca-central-1", "c5.xlarge", "alinux2", "awsbatch")
+@pytest.mark.dimensions("ca-central-1", "c5.xlarge", "awsbatch", "sge")
```

Comment:
?? :)

Reply:
oh, to hurry when change it back after testing :) fixed